src/rpm/systemd-update-helper.in

```diff
@@ -30,7 +30,8 @@ case "$command" in
         [ -d /run/systemd/system ] || exit 0
 
         users=$(systemctl list-units 'user@*' --legend=no | sed -n -r 's/.*user@([0-9]+).service.*/\1/p')
-        for user in $users; do
+        for id in $users; do
+            user=$(id -nu "$id")
```

Comment:
shellcheck will complain about this. Please use `user="$(…)"`.
(I use: `ninja -C build && shellcheck build/src/rpm/systemd-update-helper`.)

If the name cannot be resolved, this will abort the loop. I think it'd be better to skip the id and continue the loop instead.